### PR TITLE
feat: Implement new commands for databend-meta.

### LINKED ISSUE #6084: databend-meta adds cli command to send RPC to a running meta cluster
E.g.:
```shell
databend-meta --grpc-api-address 1.2.7.1:8888 --cmd kvapi::upsert --key foo --value bar
# <result in json>

databend-meta --grpc-api-address 1.2.7.1:8888 --cmd kvapi::get --key foo

databend-meta --grpc-api-address 1.2.7.1:8888 --cmd kvapi::mget --key foo bar wow

databend-meta --grpc-api-address 1.2.7.1:8888 --cmd kvapi::list --prefix foo/
```

### PR DESCRIPTION
**Pull request by RinChanNOWWW**
Signed-off-by: RinChanNOWWW <hzy427@gmail.com>

I hereby agree to the terms of the CLA available at: https://databend.rs/dev/policies/cla/

## Summary

Summary about this PR

Fixes #6084

### CODE REVIEW

**Comment by Xuanwo** [comment]
I have updated the title of PR and invited @drmingdrmer @lichuang for review, have a nice day!

File: metasrv/src/cmd/kvapi.rs [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+        let res = self.client.prefix_list_kv(self.prefix.as_str()).await?;
+        Ok(serde_json::to_string_pretty(&res)?)
+    }
+}
+
+pub struct KvApiCommand {
+    kv_op: Box<dyn KvOp>,
+}
+
+impl KvApiCommand {
```
Comment (drmingdrmer) [inline metasrv/src/cmd/kvapi.rs:98]: There is no need to introduce these structs for this task.
A function is quite enough.

trait object such as `dyn KvOp` is hard to understand and debug. Avoid using trait object, use generic type parameters or enum variants if possible.

Avoid using ErrorCode everywhere, define a dedicated error type for your crate or your mod.

Comment (Xuanwo) [reply metasrv/src/cmd/kvapi.rs:98]: > Avoid using ErrorCode everywhere, define a dedicated error type for your crate or your mod.

Nice start!

Comment (RinChanNOWWW) [reply metasrv/src/cmd/kvapi.rs:98]: > 

OK.

Comment (RinChanNOWWW) [reply metasrv/src/cmd/kvapi.rs:98]: > There is no need to introduce these structs for this task. A function is quite enough.
> 
> trait object such as `dyn KvOp` is hard to understand and debug. Avoid using trait object, use generic type parameters or enum variants if possible.
> 
> Avoid using ErrorCode everywhere, define a dedicated error type for your crate or your mod.

Ok.

**Comment by BohuTANG** [comment]
@mergify update

File: metasrv/src/cmd/kvapi.rs [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+use common_meta_types::UpsertKVReq;
+
+use crate::configs::Config;
+
+pub enum KvApi {
+    Get(String),
+    Upsert(UpsertKVReq),
+    MGet(Vec<String>),
+    List(String),
+}
```
Comment (drmingdrmer) [inline metasrv/src/cmd/kvapi.rs:30]: What about `KVApiCommand`?

File: metasrv/src/cmd/kvapi.rs [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                let res = client.mget_kv(keys.as_slice()).await?;
+                serde_json::to_string_pretty(&res)?
+            }
+            KvApi::List(prefix) => {
+                let res = client.prefix_list_kv(prefix.as_str()).await?;
+                serde_json::to_string_pretty(&res)?
+            }
+        };
+        Ok(res_str)
+    }
```
Comment (drmingdrmer) [inline metasrv/src/cmd/kvapi.rs:81]: There is no need to split command parsing and command execution into two parts. For such a task that is not very large. Just do it as simply as possible.

**Review by drmingdrmer — COMMENTED** [review]
Similar to `tests/metactl/test-metactl.sh`, these new features need a CLI test for them.

File: metasrv/src/cmd/kvapi.rs (hunk 2 above)
Comment (RinChanNOWWW) [reply metasrv/src/cmd/kvapi.rs:30]: Fine.

File: metasrv/src/cmd/kvapi.rs (hunk 3 above)
Comment (RinChanNOWWW) [reply metasrv/src/cmd/kvapi.rs:81]: Ok.

Comment (RinChanNOWWW) [reply metasrv/src/cmd/kvapi.rs:81]: So the enum is also no need.

Comment (RinChanNOWWW) [reply metasrv/src/cmd/kvapi.rs:81]: I can use this two functions in another function.

**Review by drmingdrmer — APPROVED** [review]

**Comment by drmingdrmer** [comment]
@mergify update

**Review by Xuanwo — APPROVED** [review]

**Comment by BohuTANG** [comment]
@soyeric128 databend-meta documents need update, thanks.

**Pull request merged**